Documentation/platforms/arm/imxrt/boards/imxrt1170-evk/index.rst

```diff
@@ -0,0 +1,165 @@
+===============
+i.MX RT1170 EVK
+===============
+
+`i.MX RT1170 EVK <https://www.nxp.com/design/development-boards/i-mx-evaluation-and-development-boards/mimxRT1170-evk-i-mx-RT1170-evaluation-kit:MIMXRT1170-EVK>`_
+is an evaluation kit by NXP company. This kit uses the i.MX RT1170 crossover MCU with ARM Cortex M7 core.
+
+Features
+========
+
+- Processor
+    - MIMXRT1176DVMAA processor
+    - 1GHz Cortex-M7 
+    - 400Mhz Cortex-M4
+- Memory
+    - 2 Mb OCRAM memory
+    - 512 Mbit SDRAM memory
+    - 512 Mbit Hyper Flash - Populated but 0 ohm DNP
+    - 64 Mbit QSPI Flash
+    - TF socket for SD card
+- Display and Audio
+    - MIPI LCD connectors
+- Connectivity
+    - Micro USB host and OTG connectors
+    - Ethernet (10/100T) connector
+    - Ethernet (10/100/1000T) connector
+    - CAN transceivers
+    - Arduino® interface
+- Sensors
+    - FXOS8700CQ 6-Axis Ecompass (3-Axis Mag, 3-Axis Accel)
+
+Serial Console
+==============
+
+Virtual console port provided by OpenSDA:
+
+========= ========== ==========
+UART1_TXD GPIO_AD_24 LPUART1_TX
+UART1_RXD GPIO_AD_25 LPUART1_RX
+========= ========== ==========
+
+Arduino RS-232 Shield:
+
+=== == ======= =============== ==========
+J22 D0 UART_RX GPIO_DISP_B2_11 LPUART2_RX
+J22 D1 UART_TX GPIO_DISP_B2_10 LPUART2_TX
+=== == ======= =============== ==========
+
+J-Link External Debug Probe
+===========================
+
+Install the J-Link Debug Host Tools and make sure they are in your search path.
+
+Attach a J-Link 20-pin connector to J1. Check that jumpers J5, J6, J7 and J8 are
+off (they are on by default when boards ship from the factory) to ensure SWD
+signals are disconnected from the OpenSDA microcontroller.
+
+Configurations
+==============
+
+knsh
+----
```

Comment:
Have you managed to run the protected mode configuration? I remember trying it few weeks earlier on imxrt1060 and I was not able to get it running although following the exact steps in imxrt1060 manual (which are the same as here).

Reply:
Hmm, seems there is something wrong on kernel level. I will do some more tests later and start an issue if I discover something.